src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/ActionSidebarContent.tsx

```diff
@@ -316,16 +314,13 @@ const ActionSidebarContent: FC<ActionSidebarContentProps> = ({
                 include: [GetTotalColonyActionsDocument],
               });
             }
+
             /**
-             * We need to remove all getDomainBalance queries once a payment has been completed successfully
+             * We need to remove all getDomainBalance queries once a payment has been successfully completed
+             * By default it will refetch all active queries
              */
-            client.cache.evict({
-              fieldName: 'getDomainBalance',
-            });
-            if (isQueryActive('GetDomainBalanceQuery')) {
-              client.refetchQueries({
-                include: [GetDomainBalanceDocument],
-              });
+            if (!isMotion && !isMultiSig) {
```

Comment:
So since we do this for motion and multisig finalization, what if we remove the cache entry just for payment actions? so maybe we can do it in the sagas or in the forms like this?
![image](https://github.com/user-attachments/assets/8af81ae6-33b0-43da-88b9-11739adb9da1)